scripts/deps-ppa/static_z3.sh

```diff
@@ -80,7 +80,9 @@ Vcs-Browser: https://github.com/Z3Prover/z3
 
 Package: libz3-static-dev
 Section: libdevel
-Architecture: any-i386 any-amd64
+Architecture: any-amd64
```

Comment:
We did change this in the release PPA script and our builds fail for i386 ubuntu, so we can probably drop support here as well.